### PR TITLE
Dispatch no-builds jobs to build03

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-2586

/cc @openshift/test-platform

### CODE REVIEW

File: pkg/dispatcher/config_test.go [hunk 1]
```diff
@@ -105,8 +105,9 @@ var (
 	}
 
 	configWithBuildFarmWithJobs = Config{
-		Default: "api.ci",
-		KVM:     []api.Cluster{api.ClusterBuild02},
+		Default:  "api.ci",
```
Comment (petr-muller) [inline pkg/dispatcher/config_test.go:108]: Looks like something we should clean up?

**Review by petr-muller — APPROVED** [review]
/hold
Needs a o/release followu

File: pkg/dispatcher/config_test.go (hunk 1 above)
Comment (hongkailiu) [reply pkg/dispatcher/config_test.go:108]: In the test. Any non-empty string should be fine.

**Comment by hongkailiu** [comment]
> /hold Needs a o/release followu

The breaking will be fixed after the next run of the periodic config-brancher job.

**Comment by hongkailiu** [comment]
/hold cancel
I will watch https://prow.ci.openshift.org/?job=periodic-prow-auto-config-brancher

**Comment by hongkailiu** [comment]
Manually triggered https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-prow-auto-config-brancher/1455646139192709120 which generates
https://github.com/openshift/release/pull/23226
which fixes the breaking.

**Pull request merged**